oscarpolanco/node_react_fullstack
# Issue 28: Add prettier configuration to the server directory

# Ticket or issue:

[Issue 28: Add eslint and prettier](https://github.com/oscarpolanco/node_react_fullstack/issues/28)

## Description


Add a  new prettier configuration for all the application and move the `section_text` directory

## Steps to test


- On your terminal go to the `server` directory
- Install all dependencies using: `npm install`
- Go to the `index.js` file
- Go to line 28
- Between the route and the app parameter press enter
  before => `require('./routes/surveyRoutes')(app);` 
  after =>
   ```js
   require('./routes/surveyRoutes')
   (app);
  ```
- Save the file
- If you configure `prettier` on save with your editor this should be updated as soon as you save the file. If not continue with the next step
- Go to `server` directory on your terminal
- Run the `prettier` script: `npm run prettier`
- The `index` file should return to its original state 

## Additional Notes


- In a later PR, we will run `prettier` automatically
